### ISSUE TITLE
Documentation Example is Missing modal Class

### ISSUE THREAD

**Issue by KonradHoeffner** [ghosh/Micromodal#223]
I am trying out Micromodal for the first time and the example at `https://micromodal.now.sh/#configuration` does not work because it is missing the .modal class.

I think `<div id="modal-1" aria-hidden="true">` should be `<div id="modal-1" class="modal" aria-hidden="true">`.

**Comment by ghosh** [comment]
This library is purely functional and does not make any stylistic choices for you. You are expected to style it however you need. However, to get started there is an example you can copy - https://gist.github.com/ghosh/4f94cf497d7090359a5c9f81caf60699

Closing as not a bug.
